packages/pwa-kit-create-app/scripts/build.js

```diff
@@ -23,10 +25,10 @@ const tarPathForPkg = (pkg) => p.resolve(templatesDir, `${pkg}.tar.gz`)
 
 const main = () => {
     const pkgNames = [
-        'template-retail-react-app',
-        'template-express-minimal',
-        'template-typescript-minimal',
-        'template-mrt-reference-app'
```

Comment:
I like dropping `template` in light of a future where anything can be a template

Reply:
I did this because 1. I didn't like having the template prefix in a folder named template, but it also allowed me to have the same name for the tar ball and the npm package.